README.md

```diff
@@ -154,6 +155,15 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 * [Superset](https://superset.incubator.apache.org/) - Modern, enterprise-ready business intelligence web application.
 * [Tableau](https://www.tableau.com) - Powerful and fastest growing data visualization tool used in the business intelligence industry.
 
+## Data-centric AI
+
+*Tools for improving training data to improve ML models.*
+
+* [cleanlab](https://github.com/cleanlab/cleanlab): Python library for data-centric AI and machine learning with messy, real-world data and labels.
```

Comment:
`Cleanlab` seems to fit in the `Data Validation` section.

Also, could you apply capitalization please and use a dash instead of a colon?

```suggestion
* [Cleanlab](https://github.com/cleanlab/cleanlab) - Python library for data-centric AI and machine learning with messy, real-world data and labels.
```

Reply:
applied caps and moved to data validation section